src/store/useDialogStore.ts

```diff
@@ -0,0 +1,24 @@
+import create from 'zustand';
+import { DialogStore } from '~/@types/useDialogStore';
+
+export default create<DialogStore>((set) => ({
+  title: '',
+  setTitle: (title) => {
+    set((prev) => ({ ...prev, title }));
```

Comment:
How about using `curr` instead of 'prev'?

Reply:
I think it could be both `curr` and `prev`. While it's state from previous value but will remain after reallocates.
Zustand uses `state` as variable name, so I think it'll be good to use same name.

Reply:
Good :)